ambry-cloud/src/main/java/com/github/ambry/cloud/azure/AzureCloudDestination.java

```diff
@@ -248,7 +249,7 @@ public FindResult findEntriesSince(String partitionPath, FindToken findToken, lo
    * @param blobId The {@link BlobId} to update.
    * @param fieldName The metadata field to modify.
    * @param value The new value.
-   * @return {@code true} if the udpate succeeded, {@code false} if the metadata record was not found.
+   * @return {@code true} if the udpate succeeded, {@code false} if no update was needed.
```

Comment:
udpate -> update

Was the previous javadoc inaccurate? It seems like the old code would have also thrown an exception on not found.

Reply:
yes I fixed the doc.